libsodium-sys/build.rs

```diff
@@ -415,9 +254,27 @@ fn build_libsodium() {
         };
     }
 
+    // Run ./autogen.sh
```

Comment:
If we always pin to versions from the stable branch we should never need to run autogen, which is a lot better since there are systems that don't have autoconf/automake/libtool installed.